cluster/store/db.go

```diff
@@ -22,10 +22,20 @@ import (
 	gproto "google.golang.org/protobuf/proto"
 )
 
+type (
+	applyDbOrder          bool
+	schemaCallbackTrigger bool
+)
+
 var (
 	errBadRequest = errors.New("bad request")
 	errDB         = errors.New("updating db")
 	errSchema     = errors.New("updating schema")
+
+	applyDbFirst          applyDbOrder          = true
+	applyDbLast           applyDbOrder          = false
+	triggerSchemaCallback schemaCallbackTrigger = true
+	skipSchemaCallback    schemaCallbackTrigger = false
```

Comment:
**suggestion (blocking):** i think we may don't need `applyDbLast` and `skipSchemaCallback` because they are just the negation of `applyDbFirst`, `triggerSchemaCallback` which they are bool

Reply:
The way I structured it this way is that we can understand at a glance *on the callee side* when reading the code what we are sending to apply and what it'll do.

Do you mean we could just have `applyDbLast` and `!applyDbLast` for example ? I think that can work. 

With your suggestion from above I think we can actually rely on default bool value in a struct so that we precise the other behaviour manually if we want.